packages/documentation/src/content/docs/resources/get-involved.mdx

```diff
@@ -4,27 +4,91 @@ title: Get involved
 
 import { LinkOut } from '@interledger/docs-design-system'
 
-## Participation
+Welcome to the Rafiki community! Whether you're a seasoned developer or just getting started with open source, there are many ways to contribute to the Rafiki project. This guide will help you find your place in our vibrant ecosystem.
 
-- Begin a <LinkOut href="https://github.com/interledger/rafiki/discussions">new discussion or contribute to existing discussions</LinkOut> in the Rafiki GitHub repo.
-- Join the <LinkOut href="https://interledger.slack.com">`interledger.slack.com`</LinkOut> workspace and ask any questions you have in the `#rafiki` channel.
-- Try [running the system locally](/integration/playground/overview) and ask any questions you have in the #rafiki Slack channel.
+## 🚀 Getting started
 
-For more details about spec contributions and development, visit <LinkOut href="https://github.com/interledger/rafiki/blob/main/.github/contributing.md">`contributing.md`</LinkOut>.
+### New to Rafiki?
 
-## Coding
+- **Explore the Project**: Browse the <LinkOut href="https://github.com/interledger/rafiki">Rafiki repository</LinkOut> to understand the codebase structure and recent activity.
+
+- **Join Our Community**:
+
+  - Connect with us on <LinkOut href="https://interledger.slack.com">`interledger.slack.com`</LinkOut> in the `#rafiki` channel for real-time discussions
+  - Start or join <LinkOut href="https://github.com/interledger/rafiki/discussions">GitHub Discussions</LinkOut> for deeper technical conversations
+
+- **Get Hands-On**: Try [running the system locally](/integration/playground/overview) to familiarize yourself with Rafiki's functionality. Don't hesitate to ask questions in the #rafiki Slack channel!
+
+### Make your first contribution
+
+Ready to dive in? Here's how to review good first issues if you are new to open source development.
```

Comment:
There's no mention of GitHub until the description for step 1. We should lead with that info. Maybe: Here's how to review good first issues on GitHub if you are new to open source development.